packages/notifications-flyout/src/presenters/stylesheet.js

```diff
@@ -74,7 +74,14 @@ export default function stylesheet(props, themeData) {
       } ${themeData["density.spacings.small"]}`
     },
     notificationContentText: {
-      margin: `${themeData["density.spacings.small"]}`
+      margin: `${themeData["density.spacings.small"]}`,
+      overflowWrap: `break-word`,
+      flexDirection: `row`,
```

Comment:
sorry i didn't notice this before, but there are some style here that are not necessary. it does not look like the element is a flex container so `flexDirection` and `flexWrap` wouldn't do anything here. also `flex` is shorthand for `flexShrink`, `flexGrow`, and `flexBasis`, so having flexShrink here is redundant.